front_end/html/edit_assignment.html

```diff
@@ -247,6 +247,25 @@ <h2>Create assignment</h2>
         </p>
     </div>
 
+    <div class="shadow-box">
+        <p><strong>Restrict assignment access to the following IP addresses: </strong></br>Separate valid addresses by a comma without spaces.</p>
```

Comment:
Let's avoid using <br> tags as much as possible because they do not always behave consistently. I'd have to see the user interface, but you probably don't need to put the text on a separate line. By the way, I don't think there's a `</br>` tag. I have seen it as `<br />` though.

Reply:
Okay! I was trying to match the lines in the other divs in edit_assignment.html, would you like me to remove the strong tags and line break from just the IP address line or should I make everything consistent by removing those tags from the other lines as well? Also thanks for the tip on the <br /> tag!

Reply:
I'm not sure. Can you send me an email with a screenshot so I can see how it looks?

Reply:
The screenshot looks good. Let's go with it the way you have it.